.github/workflows/update-v8.yml

```diff
@@ -0,0 +1,52 @@
+name: V8 patch update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: lts/*
+
+permissions:
+  contents: read
+
+jobs:
+  v8-update:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c  # v3.3.0
+        with:
+          persist-credentials: false
+      - uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8  # v3.3.1
+        with:
+          path: |
+            ~/.update-v8
+            ~/.npm
+        # Install dependencies
+      - name: Install Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c  # v3.6.0
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Install node-core-utils
+        run: npm install -g node-core-utils@latest
+      - name: Check and download new V8 version
+        run: |
+            ./tools/dep_updaters/update-v8-patch.sh > temp-output
+            cat temp-output
+            tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
+            rm temp-output
+      - uses: gr2m/create-or-update-pull-request-action@77596e3166f328b24613f7082ab30bf2d93079d5
+        # Creates a PR or update the Action's existing PR, or
+        # no-op if the base branch is already up-to-date.
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
+        with:
+          author: Node.js GitHub Bot <github-bot@iojs.org>
+          body: This is an automated patch update of V8 to ${{ env.NEW_VERSION }}.
+          branch: actions/update-v8-patch  # Custom branch *just* for this Action.
+          commit-message: 'deps: patch V8 to ${{ env.NEW_VERSION }}'
```

Comment:
Do we actually need this? `git node` already creates a commit with the correct title and body.
If we do, can we get the body (`Refs: ...`) back? See https://github.com/nodejs/node/commit/b8c7a1ecf8f1e9773d587f9a49b47b31e5ff11ee for an example.

Reply:
I guess we can delete the `commit-message` part